webapp/store/logic.py

```diff
@@ -525,3 +527,20 @@ def filter_charm(charm, categories=["all"], platform="all"):
         return False
 
     return True
+
+
+def get_bundle_topology(package):
+    """Retrieve and JSONifies the bundle topology data.
+    :param topology_data: Bundle topology data
+    :returns: Topology JSON
+    """
+    if package["type"] == "bundle":
+        result = yaml.load(
+            package["default-release"]["revision"]["bundle-yaml"]
+        )
+
+        topology = json.dumps(result)
+
+        return topology
+
+    return
```

Comment:
I prefer to be more explicit 
```suggestion
    return None
```